lib/rubycritic/generators/text/list.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
-require 'colorize'
+require 'rainbow/ext/string'
+
+class String
+  alias colorize color
```

Comment:
We have the (refactoring) technology, we can build him (per Working Effectively with Legacy Code).